src/main/java/com/google/sps/data/Marker.java

```diff
@@ -25,14 +25,16 @@
   private long stayMinute;
 
   private String markerName;
+  private double rating;
 
   public Marker(
       long newId,
       double newLat,
       double newLng,
       long newStayHour,
       long newStayMinute,
-      String newMarkerName) {
+      String newMarkerName,
+      double rating) {
```

Comment:
I think it may make sense to create two variables, so that you will be able to update rating when new person rates it.